### PR TITLE
Added /google

### PR DESCRIPTION
**Pull request by Ulysse2211**
Added a command for the pepole that are too lazy to google them self.

### CODE REVIEW

File: src/events/messageCreate.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ module.exports = async (client, message) => {
                 .setColor("Blurple")
                 .setDescription("👤 User - General role\n🌅 Helios User - Helios Launcher specific role")
                 .addFields({ name: "You cann't invite me on your server, but I am open source!", value: "[Source Code](https://github.com/AventiumSoftworks/Clippy)" })
```
Comment (GeekCornerGH) [inline src/events/messageCreate.js:43]:
```suggestion
                .addFields({ name: "You can't invite me on your server, but I am open source!", value: "[Source Code](https://github.com/AventiumSoftworks/Clippy)" })
```

File: src/events/messageCreate.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ module.exports = async (client, message) => {
...
-                .setFooter({ text: "Made with ❤ by GeekCornerD#8010", iconURL: "https://cdn.discordapp.com/avatars/710836174050164757/a_46c1958617a1d46fa46fab0663965ff8.gif?size=1024" })], components: [new ActionRowBuilder().addComponents([helios, user])]
+                .setFooter({ text: "Made with ❤ by GeekCornerD", iconURL: "https://cdn.discordapp.com/avatars/710836174050164757/a_46c1958617a1d46fa46fab0663965ff8.gif?size=1024" })], components: [new ActionRowBuilder().addComponents([helios, user])]
```
Comment (GeekCornerGH) [inline src/events/messageCreate.js:44]:
```suggestion
                .setFooter({ text: "Made with ❤ by GeekCorner", iconURL: "https://github.com/geekcornergh.png" })], components: [new ActionRowBuilder().addComponents([helios, user])]
```

**Review by GeekCornerGH — CHANGES_REQUESTED** [review]

**Comment by Ulysse2211** [comment]
I accepted your changes

**Review by GeekCornerGH — CHANGES_REQUESTED** [review]
Upgrade the lockfile please

**Pull request closed without merging**